### ISSUE TITLE
Fix PyPi release 

### ISSUE THREAD

**Issue by valerymelou** [valerymelou/django-active-link#126]
This error occurs when we run `make release` to publish a new version:

```
Submitting dist/django-active-link-0.2.0.tar.gz to https://upload.pypi.org/legacy/
Upload failed (400): Invalid value for blake2_256_digest. Error: Use a valid, hex-encoded, BLAKE2 message digest.
error: Upload failed (400): Invalid value for blake2_256_digest. Error: Use a valid, hex-encoded, BLAKE2 message digest.
make: *** [release] Error 1
```

Apparently we should no longer use distutils directly or even invoke setup.py manually. This [blog post](https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html) explains the recommended way.

Until this is fixed, we will not be able to release a new version of Django Active Link (see #122)

**Comment by Frodothedwarf** [comment]
Plans:
Switch the build backend to a more modern system like Poetry.
Which handles both the build part and the upload to PyPi, using a pyproject.toml file.

I think this is a smart move due to the nature of Python maturing away from distutils and setuptools.
Poetry has been around since 2018, and has proven itself to be a good choice for building the project.

**Comment by Frodothedwarf** [comment]
Futher more I suggest dropping support for old Django versions, and only support what versions are currently in Djangos lts:
![image](https://github.com/valerymelou/django-active-link/assets/48824936/2cc82fd9-7347-406e-9213-3fefd31e370f)

**Comment by Frodothedwarf** [comment]
@valerymelou everything should be fixed in #127.

If you have any changes, just reach out and I will fix them.

**Comment by valerymelou** [comment]
WoW that's an impressive work there @Frodothedwarf . I'll have a look tonight and let you know.
Many thanks

**Comment by Frodothedwarf** [comment]
@valerymelou please do not push the 0.2.0, just found something that i need to address first

**Comment by valerymelou** [comment]
Okay @Frodothedwarf waiting for your go

**Comment by Frodothedwarf** [comment]
@valerymelou it's ready now. #128 are merged and ready to be published.

**Comment by valerymelou** [comment]
This now closed in #128 
Many thanks for your help wit this @Frodothedwarf. You are a Rockstar ⭐️
